Looped/FlowControllers/PopularFlowController.swift

```diff
@@ -15,7 +14,7 @@ final class PopularFlowController: FlowController {
     }()
     
     init() {
-        navigationController.show(rootViewController, sender: nil)
+//        navigationController.show(rootViewController, sender: nil)
```

Comment:
To chyba niepotrzebne? I tak mi selectuje pierwszy tabBarItem. Co do tego delegate - dziwna sprawa. Jak robię breakpoint w Factory i sprawdzam czy jest, to potem zostaje. Jak się nie sprawdzi (po delegate) to po prostu znika. Może to mieć do czynienia z 'lazy' w rootViewController w fabryce..? (PopularFlowControllerFactory)

Reply:
Tak, niepotrzebne. Docelowo bedziemy initowac flow controller jakimis zaleznosciami